### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by ghost** [upptime/upptime#733] [Q&A]
I set-up file upptimerc.yml and show failed. Shows "Input required and not supplied: token". I completed these instructions include on your site, not helpful!
URL: https://iliya5digital.github.io/uptime/
Please help.

**Comment by AnandChowdhary** [comment]
You should remove the line https://github.com/Iliya5digital/uptime/blob/c0abc66772a2e9719a1550dda92e972723338070/.upptimerc.yml#LL25C40-L25C40 since it's not a custom domain. Let's try that and see if it works! :)
